stock_request/views/stock_request_views.xml

```diff
@@ -128,16 +128,4 @@
         </field>
     </record>
 
```

Comment:
I suggest to add to the stock.request form view the order_id field, so that a user can navigate to it.

Reply:
Done!